### PR TITLE
Remove TOC for release notes

### PR DESCRIPTION
**Pull request by alnlarsen**
We never link to this page. We forget to update it. Let's remove it.

This also removes a package.json development command which uses syntax which is not compatible with Windows

### CODE REVIEW

File: doc/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "vuepress build",
-    "dev": "NODE_ENV=development vuepress dev"
+    "dev": "vuepress dev"
```
Comment (StefanHolst) [inline doc/package.json:8]: Do we not need this anymore? If not, let's remove the use in the `.vuepress/config.js`

**Review by StefanHolst — CHANGES_REQUESTED** [review]
Minor comment

File: doc/package.json (hunk 1 above)
Comment (alnlarsen) [reply doc/package.json:8]: It doesn't work on Windows, and I got tired of removing it

Comment (alnlarsen) [reply doc/package.json:8]: It only affects the external link to the API reference. I will remove it from the config

**Review by StefanHolst — APPROVED** [review]
Looks good

**Pull request merged**